docs/writing-stories/index.md

```diff
@@ -357,7 +357,7 @@ Decorators [can be more complex](./decorators.md#context-for-mocking) and are of
 
 ## Stories for two or more components
 
-When building design systems or component libraries, you may have two or more components created to work together. For instance, if you have a parent `List` component, it may require child `ListItem` components.
+Sometimes, you may have two or more components created to work together. For instance, if you have a parent `List` component, it may require child `ListItem` components.
```

Comment:
```suggestion
Sometimes you may have two or more components created to work together. For instance, if you have a parent `List` component, it may require child `ListItem` components.
```